Tasks/ANT/task.json

```diff
@@ -120,7 +120,7 @@
             "defaultValue": "",
             "required": false,
             "groupName": "codeCoverage",
-            "helpMarkDown": "Comma separated list of filters to include or exclude classes from collecting code coverage. For example, +:com.*,+:org.*,-:my.app*.*. Refer http://www.eclemma.org/jacoco/trunk/doc/ant.html.",
+            "helpMarkDown": "Comma separated list of filters to include or exclude classes from collecting code coverage. For example, +:com.*,+:org.*,-:my.app*.*. Refer http://www.eclemma.org/jacoco/trunk/doc/ant.html for Jacoco and https://github.com/cobertura/cobertura/wiki/Ant-Task-Reference for Cobertura.",
```

Comment:
in the example the * were not showing up in UI. Might be some string formatting issues